fr.po

```diff
@@ -4379,11 +4379,11 @@ msgstr ""
 
 #. heading
 msgid "Pick Topic"
-msgstr "Choix du thème"
+msgstr "Thème"
 
 #. heading
 msgid "Pick Platform"
-msgstr "Choix de plateforme"
+msgstr "Plateforme"
```

Comment:
In fact is make too much. Usually the games are translated this way. So i tried to make it look more professionnal. The player will indeed know that he has to choose the thing.